### PR TITLE
feat: update logo

### PR DESCRIPTION
**Pull request by agarleanu**
The first commit included with this PR contains the new resources.

I think we should change the file naming and structure, but I want to hear some suggestions and feedback to what I'm about to propose.

Now that the new logo looks like this:

image

I suggest the following variants:
```
emblem
├── light
│   ├── background
│   └── transparent
└── dark
    ├── background
    └── transparent
logo
└── ...
tagline logo
└── ...
```

To anyone that's interested in contributing, feel free to comment about spacing & distances. I positioned the wordmark and the emblem as best as I could, optically, but you might have some better suggestions.

### CODE REVIEW

File: logo-slogan/biome-logo-slogan-yellow-bg.svg
Comment (Conaclos) [inline logo-slogan/biome-logo-slogan-yellow-bg.svg:1]: The image seems cropped.

**Comment by nhedger** [comment]
Thanks for the incredible work @ugudango!

I have a few suggestions:

### Convert texts to paths

It would be a good idea to convert all `text` to actual paths. If the end-user does not have the fonts used in the SVG installed on his machine, the text will fall back to a generic serif font.

Screenshot 2023-11-17 at 22 56 11

### Remove the space around the icon

Our current icon SVG has a padding around the actual logomark, but I think we should make it so that the icon touches the border of the SVG. I usually do that so it stays consistent when specifying a size with CSS. What do you think?

Screenshot 2023-11-17 at 23 00 37

Again, thanks for the hard work; really appreciate it!

**Comment by agarleanu** [comment]
Thanks for your suggestions, @nhedger ! You're right about the fonts - rookie mistake there. 😬

I think we should leave _some_ padding (like a really small one, not noticeable) since usually there's some clipping with SVGs on some odd half-pixel sizings.

I think I'll update the repo with the structure I suggested, since "yellow_background" files don't make that much sense now.

**Comment by nhedger** [comment]
> I think we should leave some padding (like a really small one, not noticeable) since usually there's some clipping with SVGs on some odd half-pixel sizing.

Interesting, I've never experienced that myself. Do you mean it happens when the SVG is given a non-integer width or height?

**Comment by Conaclos** [comment]
> Our current icon SVG has a padding around the actual logomark, but I think we should make it so that the icon touches the border of the SVG. I usually do that so it stays consistent when specifying a size with CSS. What do you think?

I understand your point, however surrounding content often assumes that the logo/image has some padding. It is the reason I added padding to the previous logo.

File: logo/biome-logo-slogan-yellow-bg.png
Comment (Conaclos) [inline logo/biome-logo-slogan-yellow-bg.png:1]: The image is not centered

**Comment by nhedger** [comment]
> > Our current icon SVG has a padding around the actual logomark, but I think we should make it so that the icon touches the border of the SVG. I usually do that so it stays consistent when specifying a size with CSS. What do you think?
> 
> 
> 
> I understand your point, however surrounding content often assumes that the logo/image has some padding. It is the reason I added padding to the previous logo.

IMHO, it makes more sense to control such padding with a container. Adding the padding directly to the SVG means that it is hardcoded, so anytime you want a different padding around the SVG, you'll have to account for the hardcoded one, which feels counterintuitive, I think.

Not a big deal overall, just thought I'd point it out so we make an informed decision.

**Comment by Conaclos** [comment]
> IMHO, it makes more sense to control such padding with a container. Adding the padding directly to the SVG means that it is hardcoded, so anytime you want a different padding around the SVG, you'll have to account for the hardcoded one, which feels counterintuitive, I think.
> 
> Not a big deal overall, just thought I'd point it out so we make an informed decision.

Otherwise, we could provide both: a padded logo and a fit one.

**Comment by nhedger** [comment]
Fair enough!

**Review by ematipico — APPROVED** [review]
I am happy merge the PR and follow up with another if there are concerns or suggestions from other people

**Pull request merged**